src/projectDB/projectversioning.cpp

```diff
@@ -18,8 +18,7 @@ namespace ProjectVersioning {
 map<Version, int> appDbVersionMap = {
     {Version("0.6.0"), 0},
     {Version("0.7.0"), 1},
-    {Version("0.8.0"), 2},
```

Comment:
the project versioning changes don't make sense to me. Why have you removed things from here?

Reply:
This is very strange. I do not remember making any of these changes. But I did resolve conflicts in this portion (using VS Code), today while rebasing on develop. Let me make see if I can undo this somehow.